public/index.html

```diff
@@ -649,19 +708,15 @@ <h2 class="i18n" data-i18n="ftm-protecting-healthcare-workers">Protecting health
             If you know construction workers, vets, scientists, or dentists, please share this link with them.
           </p>
 
-          <hr/>
-
-          <h2 class="i18n" data-i18n="ftm-add-donation-site-header">Do you need PPE?</h2>
           <div class="need-ppe">
-            <div class="i18n" data-i18n="ftm-add-donation-site-question"> Want to add a donation site? update or remove information?</div>
+            <h2 class="i18n" data-i18n="ftm-add-donation-site-header">Do you need PPE?</h2>
+            <div class="i18n" data-i18n="ftm-add-donation-site-question"><p>Want to add a donation site? update or remove information?</p></div>
             <div class="i18n" data-i18n="[html]ftm-add-donation-site-answer">
-              Please <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">
-              fill out this form</a>. We will review it as soon as possible.
+              <p>Please <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">
```

Comment:
since this is an html string you also need to update it in `en.json` to add the <p> tags. The content here is just fallback in case the correct language can't be loaded. As is this will render a <p> tag for fallback but not for other languages.

Reply:
Right. Thanks.